recipes/segemehl/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+cd segemehl
+make -j 1 ## do not use >1 make threads!
+
```

Comment:
@s-will try to set CPP_INCLUDE_PATH or related ones, not sure what segemhel honours.